mobile_verifier/src/bin/verifier_server.rs

```diff
@@ -0,0 +1,23 @@
+use mobile_verifier::{Result, Server};
```

Comment:
rust "package layout" isn't clear to me on this: will this result in two separate executable binaries, each named `verifier_server` and `verifier_generate`?

Reply:
Yeah. Any module in the `src/bin/` directory gets generated as a binary with the name of the module

Reply:
please see my previous comment.. add to the main verifier as a command instead of making a new binary please. Same way the reward server does it

Reply:
Ah, I misunderstood. Fixed.